app/controllers/users/registrations_controller.rb

```diff
@@ -43,10 +43,13 @@ def disable_account_creation
     end
 
     def track_registration(form)
-      return analytics.track_event('Account Created', form.user) unless form.email_taken?
-
-      existing_user = User.find_by_email(form.email)
-      analytics.track_event('Registration Attempt with existing email', existing_user)
+      if form.email_taken?
+        existing_user = User.find_by_email(form.email)
+        analytics.track_event('Registration Attempt with existing email', existing_user)
+      else
+        analytics.track_event('Account Created', form.user)
+        create_user_event(:account_created, form.user)
```

Comment:
Do we really need to track this? If so, since this is a one-time event, why not just read it directly from the Users table `created_at` field?